rpcs3/Emu/RSX/VK/VKTextureCache.h

```diff
@@ -434,857 +430,81 @@ namespace vk
-						}
-					}
-				}
-			}
-
-			if (!image)
-			{
-				const bool is_cubemap = type == rsx::texture_dimension_extended::texture_dimension_cubemap;
-				const VkFormat vk_format = get_compatible_sampler_format(m_formats_support, gcm_format);
-
-				image = new vk::viewable_image(*m_device, m_memory_types.device_local, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT,
-					image_type,
-					vk_format,
-					width, height, depth, mipmaps, layer, VK_SAMPLE_COUNT_1_BIT, VK_IMAGE_LAYOUT_UNDEFINED,
-					VK_IMAGE_TILING_OPTIMAL, usage_flags, is_cubemap ? VK_IMAGE_CREATE_CUBE_COMPATIBLE_BIT : 0,
-					rsx::classify_format(gcm_format));
-
-				// New section, we must prepare it
-				region.reset(rsx_range);
-				region.set_gcm_format(gcm_format);
-				region.set_image_type(type);
-				region.create(width, height, section_depth, mipmaps, image, pitch, true, gcm_format);
-			}
-
-			region.set_view_flags(swizzle_flags);
-			region.set_context(context);
-			region.set_swizzled(swizzled);
-			region.set_dirty(false);
-
-			image->native_component_map = apply_component_mapping_flags(gcm_format, swizzle_flags, rsx::default_remap_vector);
-
-			// Its not necessary to lock blit dst textures as they are just reused as necessary
-			switch (context)
-			{
-			case rsx::texture_upload_context::shader_read:
-			case rsx::texture_upload_context::blit_engine_src:
-				region.protect(utils::protection::ro);
-				read_only_range = region.get_min_max(read_only_range, rsx::section_bounds::locked_range);
-				break;
-			case rsx::texture_upload_context::blit_engine_dst:
-				region.set_unpack_swap_bytes(true);
-				no_access_range = region.get_min_max(no_access_range, rsx::section_bounds::locked_range);
-				break;
-			case rsx::texture_upload_context::dma:
-			case rsx::texture_upload_context::framebuffer_storage:
-				// Should not initialized with this method
-			default:
-				fmt::throw_exception("Unexpected upload context 0x%x", u32(context));
-			}
-
-			update_cache_tag();
-			return &region;
-		}
-
-		cached_texture_section* create_nul_section(vk::command_buffer& /*cmd*/, const utils::address_range& rsx_range, bool memory_load) override
-		{
-			auto& region = *find_cached_texture(rsx_range, { .gcm_format = RSX_GCM_FORMAT_IGNORED }, true, false, false);
-			ensure(!region.is_locked());
-
-			// Prepare section
-			region.reset(rsx_range);
-			region.set_context(rsx::texture_upload_context::dma);
-			region.set_dirty(false);
-			region.set_unpack_swap_bytes(true);
-
-			if (memory_load)
-			{
-				vk::map_dma(rsx_range.start, rsx_range.length());
-				vk::load_dma(rsx_range.start, rsx_range.length());
-			}
-
-			no_access_range = region.get_min_max(no_access_range, rsx::section_bounds::locked_range);
-			update_cache_tag();
-			return &region;
-		}
+		cached_texture_section* create_nul_section(vk::command_buffer& /*cmd*/, const utils::address_range& rsx_range, bool memory_load) override;
```

Comment:
No need to comment out the variable in the function declaration.
```suggestion
		cached_texture_section* create_nul_section(vk::command_buffer& cmd, const utils::address_range& rsx_range, bool memory_load) override;
```